### ISSUE TITLE
Materialize Docker image rejects `--log-file` option as duplicated

### ISSUE THREAD

**Issue by andrioni** [MaterializeInc/materialize#7550]
### What version of Materialize are you using?

```
materialized v0.8.3 (3e47d2809)
```

### How did you install Materialize?

* [x] Docker image

### What was the issue?
The Materialize Docker image fails to start if the `--log-file` option is passed.

```
$ docker run -it --rm materialize/materialized  --log-file /var/log/materialize.log
error: The argument '--log-file <PATH>' was provided more than once, but cannot be used multiple times

USAGE:
    materialized [OPTION]...

For more information try --help
```

### Is the issue reproducible? If so, please provide reproduction instructions.
Run
```
$ docker run -it --rm materialize/materialized  --log-file /var/log/materialize.log
```

### Please attach any applicable log files.
See above.

This is overrideable by setting a custom entrypoint, but it was still surprising behavior to me. This also means AFAIK that the Docker container is going to ignore the related environment variable, since as far as I know the CLI options have priority.

**Comment by benesch** [comment]
Ah, interesting. I suppose we could use the env var in our container instead?

**Comment by benesch** [comment]
Fixed by #10861.
